ShoppingCart/ShoppingCart/Discounts.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace ShoppingCart
+{
+    public class Discounts
+    {
+        public Dictionary<string, double> discountTable { get; private set; }

```

Comment:
Naming Violation : Public properties/variables must be named in Pascal case.
suggestion : discountTable - > DiscountTable